src/Tizen.NUI/src/public/Layouting/RelativeLayout.cs

```diff
@@ -23,85 +23,85 @@ namespace Tizen.NUI
     /// <summary>
     /// RelativeLayout calculates the size and position of all the children based on their relationship to each other.
     /// </summary>
-    [EditorBrowsable(EditorBrowsableState.Never)]
+    /// <since_tizen> 9 </since_tizen>
     public partial class RelativeLayout : LayoutGroup
     {
         /// <summary>
         /// LeftTargetProperty
         /// </summary>
-        [EditorBrowsable(EditorBrowsableState.Never)]
+        /// <since_tizen> 9 </since_tizen>
```

Comment:
I wonder if we need to open bindable properties.

Reply:
Oh, we don't need to open bindable properties..!
Patch has been updated to remove bindable properties ACR.

Reply:
Thanks!